### ISSUE TITLE
[Design] Auth Profile - Personal Info Revision - Submit design system bug reports

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#34185]
## Background

Design and Accessibility have advised that best practice is for open text fields associated with a list of radio buttons or checkboxes should be visible by default.   The radio button set components and checkbox set components in the design system do not currently function in this manner, and has been identified as a bug.  This ticket is to submit bug reports to have these components updated to function according to best practices.

## Tasks
- [x] Submit bug report to design system team for radio button set open text field functionality
- [x] Submit bug report to design system team for checkbox set set open text field functionality

## Acceptance Criteria
- [x] Bug report has been submitted to design system team for radio button set open text field functionality
- [x] Bug report has been submitted to design system team for checkbox set set open text field functionality
- [x] Both GitHub ticket numbers are added as comments in this ticket for traceability

**Comment by cgednalske** [comment]
@MarciMcGuireGCIO Bug reports submitted. Ticket numbers are [#604](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/604) and [#605](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/605) for visibility.

**Comment by MarciMcGuireGCIO** [comment]
Both tickets have been submitted to the design system team.
